highfidelity/hifi
# Case 21899: New Audio Mute States

https://highfidelity.manuscript.com/f/cases/21899/New-Mic-Bar

# TEST PLAN
- Launch Interface in Desktop mode
- Should see a new mic bar with vertical audio level meter as noted in figma document replacing the horizontal bar
- Clicking the mic bar will still toggle mute unless push-to-talk is enabled
- bubble icon next to the mic bar should toggle privacy shield upon click
- "MUTED" should show up when you speak into your microphone while you are muted
- Enable push-to-talk in `Settings > Audio`
- When you are not pushing to talk (using T keypress), the audio level meter should not move
- When you are pushing to talk, audio level meter should move
- Switch to HMD mode
- The mic bar should be horizontal (old layout) and should function the same way as what's in the current master branch
- "MUTED" should show up close to the bottom center of your screen when you speak into your microphone while you are muted
- Open the tablet
- The mic bar with horizontal audio level meter should be replaced with the new mic bar with the same functionality as what's in desktop mode
- open the `Audio` app
- Switch the "Warn when muted in HMD" setting to off
- Talk into the microphone while you are muted
- "MUTED" text should not show up in the bottom center of the screen
